### PR TITLE
Irshad plano

### PR DESCRIPTION
**Pull request by ahmi01**

### CODE REVIEW

**Comment by ahmi01** [comment]
Leave a general comment

**Comment by QikeLi** [comment]
Plano seems a city worth visiting.

File: irshad-plano.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Plano
+
+#### Places to Eat
```
Comment (QikeLi) [inline irshad-plano.md:3]: great list. awesome food

File: irshad-plano.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Plano
```
Comment (ahmi01) [inline irshad-plano.md:1]: Would you recommend visiting this place?

Thanks for informing

File: irshad-plano.md (hunk 1 above)
Comment (ahmi01) [reply irshad-plano.md:3]: Thanks QikeLiLeave a comment

**Review by ahmi01 — COMMENTED** [review]
This is great work! Go ahed and :ship:

**Pull request merged**